sparse/tests/test_coo.py

```diff
@@ -815,10 +809,24 @@ def test_gt():
     (slice(2, 0, -1), slice(None, None), -1),
     (slice(-2, None, None),),
     (slice(-1, None, None), slice(-2, None, None)),
```

Comment:
Does your new code add any new functionality?  For example are there types of slices that we now support that we didn't before?

Reply:
No, it doesn't at the moment.